test/verify/check-kdump

```diff
@@ -417,7 +420,8 @@ class TestKdumpNFS(KdumpHelpers):
         self.machines["nfs"].execute("mkdir -p /srv/kdump/var/crash; firewall-cmd --add-service nfs; systemctl restart nfs-server")
 
         # set up client machine
-        m.execute("systemctl disable kdump")
+        if m.image.startswith("fedora"):
+            m.execute("systemctl disable kdump")
```

Comment:
This feels a bit arbitrary. I thought Fedora already doesn't enable kdump.service by default?

Reply:
Dropped it in the follow up commit as it is indeed useless.